### ISSUE TITLE
Consider adding neurotree nodes based on publication info alone

### ISSUE THREAD

**Issue by stripathy** [neuroelectro/neuroelectro_neurotree#4]
Someone who needs a neurotree node but doesn't yet have one is Peter Carlen
http://www.ncbi.nlm.nih.gov/pubmed/?term=Carlen%20PL[Author]&cauthor=true&cauthor_uid=10924671

I can put together a more comprehensive list of such authors missing neurotree nodes.

**Comment by svdavid** [comment]
Sounds great.  I made some dumb moves that don't make this trivially easy. eg, now, I'm only scanning in first and middle initials from PubMed.  And not dealing with institutional affiliation, which is a kind of ugly parsing job in pub med.  Do you have any resources that could help with this?

Certainly, if you have a spreadsheet of names, it'd be trivial to import them!

**Comment by stripathy** [comment]
Certainly, if you have a spreadsheet of names, it'd be trivial to import them!
OK - I'll do the following: provide @svdavid a spreadsheet with a list of authors (and pmids) with no corresponding neurotree node. We can decide later whether to import them or not.
I'll also provide a list of authors who are in neurotree but are effectively unconnected from the rest of the tree due to edge incompleteness.
